recipe/meta.yaml

```diff
@@ -1,54 +1,42 @@
 {% set name = "graphql-core" %}
-{% set version = "1.1" %}
-{% set bundle = "tar.gz" %}
+{% set version = "2.0" %}
 {% set hash_type = "sha256" %}
-{% set hash = "63bb8593aeeadb0a53e14207b910027fe51158d017927fad87326dac806185ee" %}
-{% set build = 1 %}
+
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4830699be53f9154273fa15726fc8b0c90bc22bbb8fc7c932586503b3cb9330e
 
 build:
   noarch: python
-  number: {{ build }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
-  build:
+  host:
     - python
-    - setuptools
-
+    - pip
   run:
     - python
     - six >=1.10.0
-    - promise >=2.0
+    - promise >=2.1
+    - rx >=1.6.0
 
 test:
   imports:
     - graphql
-    - graphql.error
```

Comment:
I see no reason to drop these, since there are no additional tests...

Reply:
The reason is that all relevant symbols are imported here https://github.com/graphql-python/graphql-core/blob/master/graphql/__init__.py so there is no point in explicitly testing the single imports other than `import graphql`.

Reply:
I take the liberty to merge this anyway so that we land the package update (looks like we have time-zone issues 🌞 🌑 😉). In case you are not ok with the new tests you can open an issue or a PR and we can discuss it there.